src/app/navigation/pagination.component.ts

```diff
@@ -113,24 +114,37 @@ export class PaginationComponent {
      *          select option which holds current value selected.
      */
     updateLimit(option) {
-        // when selecting a limit greater than the total the following exception is thrown
-        // throws browser_adapter.js:76 EXCEPTION: Attempt to use a dehydrated detector: PaginationComponent_1 -> ngModelChange
+        // occasionally throws browser_adapter.js:76 EXCEPTION: Attempt to use a dehydrated detector: PaginationComponent_1 -> ngModelChange
         // this seems to be an issue with Angular2 and/or Angular Universal
+        // I cannot infer what is causing the problem from the following code.
```

Comment:
Not a fix, but a workaround:
https://github.com/angular/angular/issues/6786
Wrap the 'this.router.navigate' call in a setTimeout().

I tested this locally and it seemed to resolve the issue. Albeit in a bit of an ugly way :-)